extract_endpoint/src/azure_utils.py

```diff
@@ -0,0 +1,38 @@
+import enum
+from io import BytesIO
+from typing import Optional, NamedTuple
```

Comment:
prefer `import typing`, `import io`, etc.

Reply:
👍  on io, but I changed `import typing` to `from typing import Optional, NamedTuple` to fix a pylint bug (see https://github.com/PyCQA/pylint/issues/1418) :(